venter/views.py

```diff
@@ -7,13 +7,14 @@
 from venter.models import Complaints
 from venter.models import Comment
 from venter.models import ComplaintMedia
-from venter.models import  TagUris
+from venter.models import TagUris
 
 from venter.serializers import ComplaintSerializer
 from venter.serializers import ComplaintPostSerializer
 from venter.serializers import CommentPostSerializer
 from venter.serializers import CommentSerializer
 
+
```

Comment:
Whitespace change. You can view these on your command prompt before you commit by `git diff`